cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_svd.cpp

```diff
@@ -14,17 +14,101 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include <daal/src/algorithms/pca/pca_dense_svd_batch_kernel.h>
+#include <daal/include/algorithms/normalization/zscore_types.h>
+
 #include "oneapi/dal/algo/pca/backend/cpu/train_kernel.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/error_converter.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+#include "oneapi/dal/table/row_accessor.hpp"
 
 namespace oneapi::dal::pca::backend {
 
+using std::int64_t;
```

Comment:
Let's try avoid usings from `std`, I now, we used that earlier, but in new code we try to avoid.

Reply:
fixed